calico_mesos/calico_mesos.py

```diff
@@ -104,6 +105,10 @@ def _setup_logging(logfile):
     handler.setFormatter(formatter)
     _log.addHandler(handler)
 
+    pycalico_logger = logging.getLogger(pycalico.__name__)
+    pycalico_logger.setLevel(logging.DEBUG)
```

Comment:
I'm not sure we want to be logging at DEBUG without some way to modify the level at runtime.